requirements.bazel.txt

```diff
@@ -0,0 +1,16 @@
+# GRPC Python setup requirements
```

Comment:
How do we keep this up to date now?

Reply:
Up to date requirements might require Python later versions which is incompatible with ESP build rules.

Currently it's pinned to a specific version which could not update. How do you think we should address this?

Reply:
I don't know. I think we have no choice to postpone the tech debts to later. Can you please log a TODO bug? File one in bugnizer so we can ask for resources based on these data points.

Reply:
Sure, created todo and internal bug b/341556079.